src/e2e-tests/src/intTest/resources/behavior/ui/csAddMember.feature

```diff
@@ -8,6 +8,9 @@ Feature: Central Server Add Member
     And User xrd logs in to CentralServer with password secret
 
   Scenario: Member is created and listed
-    Given Members tab is selected
-    When A new member is added
+    Given CentralServer Settings tab is selected
+    And System settings tab is selected
+    And A new memberclass XXX is added
```

Comment:
XXX is bit confusing and will be hard to search. Lets make it clear that this came from test automation.

We've also discussed making it dynamic, but we can leave it as is for now.